packages/node_modules/@ciscospark/internal-plugin-metrics/src/call-diagnostic-events-batcher.js

```diff
@@ -0,0 +1,42 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import Batcher from './batcher';
+
+const CallDiagnosticEventsBatcher = Batcher.extend({
+  namespace: `Metrics`,
+
+  prepareItem(item) {
+    // Add buildType and networkType
+    const origin = {
+      buildType: process.env.NODE_ENV === `production` ? `prod` : `test`,
+      networkType: `unknown`
```

Comment:
networkType: 
enum:
- wifi
 - ethernet
- cellular
- unknown

I think we cannot get this info from browsers right now. However, it is a required field.